authlib/integrations/base_client/registry.py

```diff
@@ -86,6 +86,16 @@ def register(self, name, overwrite=False, **kwargs):
         self._registry[name] = (overwrite, kwargs)
         return self.create_client(name)
 
+    def remove_client(self, name):
+        if name not in self._registry:
+            raise KeyError(f'Client {name} not found in registry.')
+
+        if name not in self._clients:
+            raise KeyError(f'Client {name} not found in clients.')
+
+        del self._registry[name]
+        del self._clients[name]
```

Comment:
_Hello @lcdunne,_

This might be implemented in a bit cleaner and safer way.
```suggestion
        missing = []

        if not self._registry.pop(name, None):
            missing.append['registry']
        if not self._clients.pop(name, None):
            missing.append['clients']

        if missing:
            raise KeyError(f'Client {name} not found in f{" nor ".join(missing)}.')
```

_Best regards!_